.envrc.example

```diff
@@ -0,0 +1,6 @@
+export SOURCE_WIKI_URL=''
```

Comment:
I think if we prefer to pass these as env vars instead of positional args, we should do it in the containerized script instead. Else we'll end up with two different versions of this API, which can cause confusion.